src/EditorFeatures/CSharp/NavigationBar/CSharpNavigationBarItemService.cs

```diff
@@ -346,12 +346,6 @@ private static void ValidateSpanFromBounds(ITextSnapshot snapshot, int start, in
             Contract.Requires(start >= 0 && end <= snapshot.Length && start <= end);
         }
 
-        [Conditional("DEBUG")]
-        private static void ValidateSpan(ITextSnapshot snapshot, int start, int length)
```

Comment:
Yes I built with debug, the analysis tool used needs a little work on it to select build flavours to ensure that when it uses Roslyn to create the semantic model that all the correct options are set.